tests/Linter/LintingExceptionTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Linter;
+
+use PhpCsFixer\Linter\LintingException;
+use PHPUnit\Framework;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Console\Command\DescribeNameNotFoundException
+ */
+final class LintingExceptionTest extends Framework\TestCase
```

Comment:
Please add a `testConstructorSetsValues` test here and in `UnavailableLinterExceptionTest`: those exceptions are not marked as `@internal` so the tests should guaranty that the constructor does not break (even if it is the inherited constructor).